build.gradle.kts

```diff
@@ -104,49 +101,50 @@ bukkit {
         "Plugin that provides functionality of parcel lockers in Minecraft, allowing players to send and receive parcels safely."
     website = "https://github.com/EternalCodeTeam/ParcelLockers"
     version = "1.0.0-SNAPSHOT"
+    depend = listOf("Vault")
 }
 
 tasks.withType<JavaCompile> {
     options.encoding = "UTF-8"
     options.setIncremental(true)
 }
 
-tasks.withType<ShadowJar> {
-    archiveFileName.set("ParcelLockers v${project.version} (MC 1.8.8-1.20.x).jar")
-
-    exclude(
-        "org/intellij/lang/annotations/**",
-        "org/jetbrains/annotations/**",
-        "META-INF/**",
-        "javax/**",
-    )
-
-    mergeServiceFiles()
-    minimize()
-
-    val prefix = "com.eternalcode.parcellockers.libs"
-    listOf(
-        "panda",
-        "org.panda_lang",
-        "net.dzikoysk",
-        "io.papermc.lib",
-        "org.bstats",
-        "dev.rollczi",
-        "net.kyori",
-        "okhttp3",
-        "org.json",
-        "com.fasterxml"
-    ).forEach { pack ->
-        relocate(pack, "$prefix.$pack")
-    }
-}
-
 tasks {
     runServer {
         minecraftVersion("1.20.1")
     }
-}
 
-tasks.getByName<Test>("test") {
-    useJUnitPlatform()
+    test {
+        useJUnitPlatform()
+    }
+
+    shadowJar {
+        archiveFileName.set("ParcelLockers v${project.version} (MC 1.8.8-1.20.x).jar")
+
+        exclude(
+            "org/intellij/lang/annotations/**",
+            "org/jetbrains/annotations/**",
+            "META-INF/**",
+            "javax/**",
+        )
+
+        mergeServiceFiles()
+        minimize()
+
+        val prefix = "com.eternalcode.parcellockers.libs"
+        listOf(
+            "panda",
+            "org.panda_lang",
+            "net.dzikoysk",
+            "io.papermc.lib",
+            "org.bstats",
+            "dev.rollczi",
+            "net.kyori",
+            "okhttp3",
```

Comment:
Why you using okhttp3 in minecraft plugin? (not related to review, just question)